apps/design-system/src/views/repo-commits/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { useCallback, useState } from 'react'
+
+import { BranchSelectorTab, IBranchSelectorStore, RepoCommitsView as RepoCommitsUiView } from '@harnessio/ui/views'
+
+import { noop, useTranslationsStore } from '../../utils.ts'
+import { repoFilesStore } from '../repo-files/components/repo-files-store.ts'
+import { repoCommitsStore } from './repo-commits-store.ts'
```

Comment:
We shouldn't need the `.ts` on imports
```suggestion
import { noop, useTranslationsStore } from '../../utils'
import { repoFilesStore } from '../repo-files/components/repo-files-store'
import { repoCommitsStore } from './repo-commits-store'
```